drivers/storage/portworx/manifest/manifest.go

```diff
@@ -57,6 +59,7 @@ type Release struct {
 	PrometheusOperator        string `yaml:"prometheusOperator,omitempty"`
 	PrometheusConfigMapReload string `yaml:"prometheusConfigMapReload,omitempty"`
 	PrometheusConfigReloader  string `yaml:"prometheusConfigReloader,omitempty"`
+	Telemetry                 string `yaml:"telemetry,omitempty"`
```

Comment:
Let's also create a px-installer ticket to add it there, if not already created.

Reply:
https://portworx.atlassian.net/browse/PSG-200 for next sprint.